test/PDPVerifier.t.sol

```diff
@@ -693,7 +732,21 @@ contract PDPVerifierProofTest is Test, ProofBuilderHelper {
 
         // Submit proof successfully, advancing the proof set to a new challenge epoch.
         vm.mockCall(pdpVerifier.RANDOMNESS_PRECOMPILE(), abi.encode(challengeEpoch), abi.encode(challengeEpoch));
-        pdpVerifier.provePossession{value: PDPFees.proofFee(proofs.length)}(setId, proofs);
+        // Mock Pyth oracle call to return $5 USD/FIL
+        bytes memory pythCallData = abi.encodeWithSelector(
```

Comment:
These code blocks are repeated many times.  I think you could encapsulate call data generation into a helper function of price and expo.

Reply:
Done.